EIPS/eip-5659.md

```diff
@@ -0,0 +1,47 @@
+---
+eip: 5659
+title: Social Media URI Propagation Event
+description: A minimal primitive to emit URI events for social media.
+author: Tim Daubenschütz (@TimDaub), Auryn Macmillan (@auryn-macmillan)
+discussions-to: https://ethereum-magicians.org/t/social-media-uri-propagation-event/10893
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-09-15
+---
+
+## Abstract
+
+We introduce a minimal on-chain primitive to emit URIs using Ethereum's event log infrastructure.
```

Comment:
We like terse abstracts, but this is perhaps a touch too short. Perhaps include a brief summary of what kinds of URIs you expect people to publish?